starter_code/app.js

```diff
@@ -0,0 +1,64 @@
+
+var SpotifyWebApi = require('spotify-web-api-node');
+var clientId = '2bb8357c009d414aa977da5341a6ac1a',
+    clientSecret = 'ef0d1a3f23a14ef8a5cd4934493969cd';
+var spotifyApi = new SpotifyWebApi({
+  clientId : clientId,
+  clientSecret : clientSecret
+});
+spotifyApi.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.